projects/graph/README.md

```diff
@@ -189,6 +185,6 @@ Highlight the route.
 ### Stretch 5: Travelling saleperson
 
 Read about the [travelling salesperson problem](https://en.wikipedia.org/wiki/Travelling_salesman_problem).
-This is a *hard* problem, literally, so even as a stretch goal a real
+This is a _hard_ problem, literally, so even as a stretch goal a real
 implementation is not expected - but it's an interesting and famous problem
 in the space, and worth learning about if you want to go deep in graphs.
```

Comment:
Whenever they ask questions in the README you should consider making a new files, something like a *notes.md* and take some time to research the answers. The core concepts can sometimes become more clear if you get stuck somewhere on a particularly difficult piece of code.